### ISSUE TITLE
model.predict() gives different results when batch_size =1 using Batch Norm

### ISSUE THREAD

**Issue by sameervk** [keras-team/keras#13762]
<em>Please make sure that this is a Bug or a Feature Request and provide all applicable information asked by the template.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.</em>  

**System information**  
- Have I written custom code (as opposed to using example directory):  No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Ubuntu 18.04 LTS
- TensorFlow backend (yes / no):  yes
- TensorFlow version:  2.1.0
- Keras version:  2.2.4-tf
- Python version:  3.6.7
- CUDA/cuDNN version:  
- GPU model and memory:  

You can obtain the TensorFlow version with:  
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"  
You can obtain the Keras version with:  
python -c 'import keras as k; print(k.__version__)'  

**Describe the current behavior**  
Running model.predict() on a batch gives accurate values. However, when run on a single sample, i.e. batch size 1 gives same output [8] on every individual sample of the original batch. 
**Describe the expected behavior**  
Running predict() on batch_size = 1 should not result in the above behavior
**Code to reproduce the issue**  
Testing the behaviour of batch norm layer in keras using MNIST number dataset using the model from the original BatchNorm paper. Using 3 dense layers (100 units each) with 3 adjacent bnorm layers and a final dense layer with 10 units; input flattened from 28*28 to 784; batch size = 60; no activation for the final layer. Please find attached is .h5 (fmt: model_{epoch}-{val_loss}) file for the model
[model_10-0.119.zip](https://github.com/keras-team/keras/files/4185898/model_10-0.119.zip)

**Other info / logs**  
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by jneeven** [comment]
@sameervk could you explain what the issue was? We're running into the same issue here...
